packages/App/src/components/wallet/AddMonetaryMovements.tsx

```diff
@@ -132,44 +126,42 @@ const AddMonetaryMovements: FC<Props> = ({navigation}) => {
     },
   });
 
+  const getLocation = async () => {
```

Comment:
Let's not mess `async` and Promises. It should be either Promises or `async/await`